### DISCUSSION TITLE
nilesoft.org down?

### DISCUSSION THREAD

**Discussion by Zastrutzki** [moudey/Shell#460] [General]
Time for my yearly clean install. 

Is nilesoft dead?

**Comment by stamminator** [comment]
It's been quiet for a few weeks. I hope @moudey is well. This project is still fairly young, so I'd be surprised if it's dead.

**Comment by moudey** [comment]
We are sorry about this.  The hosting company is migrating to a new server, so there are some problems, and I am in contact with support.

**Reply by Dhyfer1** [reply]
You have your repository on GitHub, why don't you upload the installer file to Releases like everyone else does?

**Comment by Zastrutzki** [comment]
It's back online! Keep up the good work, @moudey!

**Reply by Dhyfer1** [reply]
That site goes down several times and it's getting annoying, right now it's not possible to view the site 😤
